### ISSUE TITLE
createLocalTracks not finding external webcam

### ISSUE THREAD

**Issue by CourtneyBurke** [jitsi/lib-jitsi-meet#810]
I have an external webcam with a built in mic connected(Logitech), currently when calling createLocalTracks this webcam is not being picked up as a video device or an audio device. I can use this webcam when connected to my Jitsu instance in the browser, but the webcam cannot be found when calling createLocalTracks in JavaScript in my own application.

To Reproduce:
- Have an external webcam connected
- In JavaScript connect to conference, join a room and call createLocalTracks
- Only the laptops built in webcam and mic is displayed, not the connected external webcam

Environment details:
- Chromium(Chrome) Browser within an Electron App

**Comment by damencho** [comment]
So if you load meet.jit.si in the browser you can use the camera, right?
Can you try doing the same using jitsi-meet-electron, can you use it there? I suspect chrome versions, we had some problems like that in the past.

**Comment by CourtneyBurke** [comment]
So I downloaded and installed the Jitsi-meet electron app and my external Logitech webcam was available for use

**Comment by damencho** [comment]
So it doesn't work, in an application you are developing, right? What version of electron is there and what version of lib-jitst-meet?

**Comment by CourtneyBurke** [comment]
Correct, it doesn't work for me in the application I am developing.
I was on electron 1.8.4. 
Yesterday I upgraded my electron to the latest stable release which is 2.0.8, on this version I get different behaviour, upon calling createLocalTracks an error is returned straight away. The error is "name: track.no_data_from_source, message: The track has stopped receiving data from it's source"

**Comment by damencho** [comment]
Where do you get lib-jitsi-meet from?

**Comment by CourtneyBurke** [comment]
Any ideas on the latest error ?

**Comment by damencho** [comment]
This error can be caused by no data been sent when data is supposed to be sent...

**Comment by CourtneyBurke** [comment]
Data being sent between where? 
Any idea on what I can do to resolve it ?

**Comment by damencho** [comment]
What is the example that you are running?

**Comment by CourtneyBurke** [comment]
Sorry I don't understand the question

**Comment by CourtneyBurke** [comment]
I am writing my own app which is using the jitsi Js lib pointing at our own jitsi deployment.

**Comment by CourtneyBurke** [comment]
Hi, sorry for what might appear as impatience but we have a demo coming up which we want to use jitsi for and this issue is blocking us. Any ideas on what causes the error

**Comment by bbaldino** [comment]
Can you provide the exact model of the webcam?

**Comment by bbaldino** [comment]
Also, can you visit this page using the webcam and tell us what you see: https://webrtchacks.github.io/WebRTC-Camera-Resolution/

**Comment by damencho** [comment]
@CourtneyBurke is this the only camera that doesn't work? Does it work with others?

**Comment by CourtneyBurke** [comment]
I dont think it is a webcam/rtc issue, this same webcam worked on the jitsi test app and on the jitsi electron app

**Comment by CourtneyBurke** [comment]
Tried the full scan on the webrtc link you sent again and it worked, I can see my camera feed, I just had to allow access to my webcam on the browser.
